rules/sonic-platform-common.dep

```diff
@@ -0,0 +1,21 @@
+
+SPATH       := $($(SONIC_PLATFORM_COMMON_PY2)_SRC_PATH)
+DEP_FILES   := $(SONIC_COMMON_FILES_LIST) rules/sonic-platform-common.mk rules/sonic-platform-common.dep   
+DEP_FILES   += $(SONIC_COMMON_BASE_FILES_LIST)
+SMDEP_FILES := $(addprefix $(SPATH)/,$(shell cd $(SPATH) && git ls-files|grep -Ev "sonic_sfp|sonic_eeprom"))
```

Comment:
why is this special?

Reply:
sonic_sfp and sonic_eeprom are created as soft links. The git hash-object command is not working with soft links. Hence these files are excluded.

Reply:
@jleveque , why do we have soft links? can we remove them?